algorithms/ar-bsrh/python2/binary_search.py

```diff
@@ -0,0 +1,27 @@
+def binarysearch(arr,x):
+  low=0
+  high=len(arr)-1
+  mid=0
+  while(low<=high): 
+    mid=(low+high)//2
+    if(arr[mid]<x):
+      low=mid+1
+    elif(arr[mid]>x):
+      high=mid-1
+    else: 
```

Comment:
Codacy found an issue: [Trailing whitespace](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6277716)

Reply:
this is also the same mistake.